compiler/rustc_parse/src/lexer/tokentrees.rs

```diff
@@ -98,7 +98,7 @@ impl<'a> TokenTreesReader<'a> {
         err
     }
 
-    fn parse_token_tree_open_delim(&mut self, open_delim: Delimiter) -> TokenTree {
+    fn parse_token_tree_open_delim(&mut self, open_delim: Delimiter) -> PResult<'a, TokenTree> {
```

Comment:
Nit: changes in this file do not logically belong to this commit.

Reply:
I split the changes in this file to the second commit.